moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -4,6 +4,8 @@ import { Link as RouterLink, useParams, useLocation } from "react-router-dom";
 import { createBrowserHistory } from "history";
 import { makeStyles } from "@material-ui/core/styles";
 
+import KnackSync from "./KnackSync.js";
+
```

Comment:
Another small convention, we don't include the `.js` part of the file name

Reply:
Got it - thanks for pointing that out.